lib/Field/TextField.php

```diff
@@ -1,17 +1,25 @@
 <?php
 
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2016 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
 namespace Symfony\Cmf\Component\ContentType\Field;
 
-use Symfony\Component\OptionsResolver\OptionsResolver;
 use Symfony\Cmf\Component\ContentType\FieldInterface;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
-use Symfony\Cmf\Component\ContentType\View\ScalarView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class TextField implements FieldInterface
 {
     public function getViewType()
     {
-        return ScalarView::class;
+        return 'scalar';
```

Comment:
In keeping with the precedent now set by the form framework, we should probably be using the FQN of the type as previously.